// @openzeppelin/truffle-upgrades/dist/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
var __exportStar = (this && this.__exportStar) || function(m, exports) {
    for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.beacon = exports.erc1967 = exports.admin = exports.upgradeBeacon = exports.deployBeaconProxy = exports.deployBeacon = exports.silenceWarnings = void 0;
var upgrades_core_1 = require("@openzeppelin/upgrades-core");
Object.defineProperty(exports, "silenceWarnings", { enumerable: true, get: function () { return upgrades_core_1.silenceWarnings; } });
__exportStar(require("./deploy-proxy"), exports);
__exportStar(require("./prepare-upgrade"), exports);
__exportStar(require("./upgrade-proxy"), exports);
var deploy_beacon_1 = require("./deploy-beacon");
Object.defineProperty(exports, "deployBeacon", { enumerable: true, get: function () { return deploy_beacon_1.deployBeacon; } });
var deploy_beacon_proxy_1 = require("./deploy-beacon-proxy");
Object.defineProperty(exports, "deployBeaconProxy", { enumerable: true, get: function () { return deploy_beacon_proxy_1.deployBeaconProxy; } });
var upgrade_beacon_1 = require("./upgrade-beacon");
Object.defineProperty(exports, "upgradeBeacon", { enumerable: true, get: function () { return upgrade_beacon_1.upgradeBeacon; } });
var admin_1 = require("./admin");
Object.defineProperty(exports, "admin", { enumerable: true, get: function () { return admin_1.admin; } });
var erc1967_1 = require("./erc1967");
Object.defineProperty(exports, "erc1967", { enumerable: true, get: function () { return erc1967_1.erc1967; } });
var beacon_1 = require("./beacon");
Object.defineProperty(exports, "beacon", { enumerable: true, get: function () { return beacon_1.beacon; } });
